server/src/test/java/io/crate/expression/scalar/HasDatabasePrivilegeFunctionTest.java

```diff
@@ -155,13 +158,21 @@ public void test_create_privilege() {
     public void test_same_results_for_name_and_oid() {
         int dbOid = Constants.DB_OID;
         int userOid = OidHash.userOid("test");
-        // Testing all 6 possible signatures.
+        // Testing all 6 possible signatures, for a normal user but also for superuser.
+        assertEvaluate("has_database_privilege('crate', 'crate', 'CREATE')", true);
```

Comment:
don't we need  `sqlExpressions = new SqlExpressions(tableSources, null, User.CRATE_USER);`
before testing on a superuser? in @Before section user is selected randomly, can be flaky?

Reply:
If the session user has cluster privileges then it can check the privileges of the super user as well, so all of the users in `@Before` can do that, and check the privileges of `crate` user.

Reply:
thx for clarifying!